src/components/YourFeed.tsx

```diff
@@ -243,13 +245,16 @@ export class YourFeed extends React.PureComponent<DispatchProps & StateProps, Yo
         const url = post.link || '';
         const hostnameText = url === '' ? '' : ' -  ' + Utils.getHumanHostname(url);
         return (
-            <View style={styles.infoContainer}>
+            <TouchableOpacity
+                onPress={() => this.props.navigation.navigate('Feed', { feedUrl: post.author && post.author.uri })}
```

Comment:
I think it would be better to pass `post.author` here and then we could display the name of the feed as the title of the navigation header.

Reply:
good idea